README.md

```diff
@@ -26,3 +26,21 @@ Debian build facilities are integrated into the repo at `debian/`.
 3. Install the resulting `jellyfin*.deb` file on your system.
 
 A huge thanks to Carlos Hernandez who created the Debian build configuration for Emby 3.1.1.
+
+### Windows (64 bit)
+A pre-built windows installer will be available at [The JellyFin Repository](https://repo.jellyfin.org/).
+
+1. Install the dotnet core SDK 2.1 from [Microsoft's Webpage](https://dotnet.microsoft.com/download/thank-you/dotnet-sdk-2.1.500-windows-x64-installer)
```

Comment:
Hmm, the link looks a bit strange. Also I think we'd better keep generic 2.1 link, rather than specific 2.1.500 version.

Reply:
Agreed, as mentioned below, link to https://dotnet.microsoft.com/download/dotnet-core/2.1 instead to be general.